javaslang/src-gen/main/java/javaslang/Tuple.java

```diff
@@ -213,4 +216,73 @@ static Tuple0 empty() {
     static <T1, T2, T3, T4, T5, T6, T7, T8> Tuple8<T1, T2, T3, T4, T5, T6, T7, T8> of(T1 t1, T2 t2, T3 t3, T4 t4, T5 t5, T6 t6, T7 t7, T8 t8) {
         return new Tuple8<>(t1, t2, t3, t4, t5, t6, t7, t8);
     }
+
+    static <T1> Tuple1<Seq<T1>> sequence1(Iterable<? extends Tuple1<? extends T1>> tuples) {
```

Comment:
why is there a need for the number at the end, it should be distinguishable via parameters?